playwright/async_api/__init__.py

```diff
@@ -76,7 +82,28 @@ def async_playwright() -> PlaywrightContextManager:
     return PlaywrightContextManager()
 
 
+@overload
+def expect(page_or_locator: Page) -> PageAssertions:
+    ...
```

Comment:
What is this? Does it mean "pass"? I am confused :)

Reply:
This is how Python handles overloads. Since the return value when you pass a page is different compared when you pass a locator.